### PR TITLE
Preserve sign of `0`.

### PR DESCRIPTION
**Pull request by Xotic750**
Part of https://github.com/lodash/lodash/issues/1574

### CODE REVIEW

File: lodash.js [hunk 1]
```diff
@@ -9760,6 +9760,10 @@
      * @returns {string} Returns the string.
      */
     function toString(value) {
+      // Preserve sign of `0`
+      if (value === 0) {
+        return 1 / value !== Infinity ? '-0' : '0';
```
Comment (jridgewell) [inline lodash.js]: Double negative... mind switching?

Comment (Xotic750) [inline lodash.js]: I have no preference. Can switch it in my next PR if wanted?

Comment (jridgewell) [inline lodash.js]: Please.

Comment (jdalton) [inline lodash.js]: Don't worry, I plan on cleaning it up tonight.

Comment (Xotic750) [inline lodash.js]: Oh well, they are in the PR.

**Pull request merged**